practices/dependency-management.md

```diff
@@ -0,0 +1,104 @@
+
+We use some [dependency management tools](#dependency-management-tools) to help us keep our apps up to date when new versions of their dependencies are released.
+
+
+### Run-time dependencies
+
+[Run-time dependencies](https://docs.npmjs.com/files/package.json#dependencies) are modules that are required for your application to run, independently of the environment or mode (e.g. development vs production) used.
+
+These are defined in the `dependencies` section of the `package.json` file.
+
+Version numbers for run-time dependencies are defined using a tilde `~` plus a full version number in the MAJOR.MINOR.PATCH form. If the PATCH version is 0, it can be omitted.
+
+Specifying the versions in this way ensures that:
+* We don't expose the users of our modules to an unnecessary risk if one of our dependencies releases a minor update that breaks our app.
+* Dependencies in early lifecycle projects (v0.MINOR.PATCH) are traditionally considered of beta quality, or undergoing heavy development, and any minor could potentially contain breaking changes. Limiting the scope to patch updates heavily reduces the chance of our app breaking.
+* The behaviour when installing the dependencies is basically as predictable as possible ([Principle of least astonishment](https://en.wikipedia.org/wiki/Principle_of_least_astonishment)) without the use of npm-shrinkwrap or similar tools.
+
+Don't use wildcards (i.e. `*`, `x`) in the package numbers or keywords like `latest`. They make the versions less clear, and predicting the outcome of installing the dependencies harder.
+
+
+#### Examples
+
+We do this:
+
+```json
+{
+    "dependencies": {
+        "airbrake": "~1.2.0",
+        "dustmite": "~1.0.0",
+        "shunter": "~4.2.1",
+        "thundermole": "~1.0.3",
+        "truffler": "~1.1"
+    }
+}
+```
+
+We _don't_ do this:
+```json
+{
+    "dependencies": {
+        "airbrake": "latest",
+        "dustmite": "*",
+        "shunter": "~4",
+        "thundermole": "^1.0.3",
+        "truffler": "~1.x"
+    }
+}
+```
+
+[Optional dependencies](https://docs.npmjs.com/files/package.json#optionaldependencies), if present, should also use the same format.
+
+
+### Development dependencies
+
+[Development dependencies](https://docs.npmjs.com/files/package.json#devdependencies) are required during the development, testing, or build process of the app. For example, you may include a test runner, a minifier, a bundler, etc.
+
+These are defined in the `devDependencies` section of the `package.json` file and are not used in production environments.
+
+Version numbers for run-time dependencies are defined using a caret `^` plus a full version number in the MAJOR.MINOR.PATCH form. If the MINOR and PATCH versions are 0, they can be omitted.
```

Comment:
I suggest removing this: "If the PATCH version is 0, it can be omitted."

Should avoid unnecessary exceptions, as it makes for simpler documentation, and in this case will result in less deviation from the norm in the code.